src/server/repositories/UserRepository.ts

```diff
@@ -160,30 +141,28 @@ export class UserRepository implements UserRepositoryInterface {
         },
       ],
     }
-    const whereConditions: any =
+    let whereConditions: any =
       conditions.searchText.length > 0
         ? {
             ...searchTextCondition,
             userId: conditions.userId,
           }
         : { userId: conditions.userId }
+    if (conditions.tags && conditions.tags.length > 0) {
+      const searchTagConditions = {
+        [Op.or]: conditions.tags.map((tag) => {
+          return { tagStrings: { [Op.substring]: `${tag}` } }
+        }),
```

Comment:
v small nit:
```suggestion
        [Op.or]: conditions.tags.map((tag) => (
          { tagStrings: { [Op.substring]: tag } }
        )),
```
also have a question: since we're searching based on `tagStrings` now (from urls) instead of `tagKey` from before (from tags), will it mean that our search is now case-sensitive?

Reply:
sorry, didn't push the latest change  😆

Reply:
the Op.iLike should be able to handle case insensitive search